1904-11-01.xml

```diff
@@ -0,0 +1,5261 @@
+                        fleet.</p>
+                </div>
+                <div type="section" feature="wire">
+                    <head>TO-DAY'S TELEGRAMS.</head>
+                    <div type="item">
+                        <head>SIGNIFICANT MOVEMENT OF JAPANESE.</head>
+                        <head>ASSUMING THE OFFENSIVE.</head>
+                        <div type="cable">
+                            <dateline>ST. PETERSBURG, October 31.</dateline>
+                            <p>General Kuropatkin reports that the Japanese have begun a significant
+                                movement by concentrating eastwards. They are receiving
+                                reinforcements, evidently preparatory to assuming the offensive.
+                                (Reuter.)</p>
+                        </div>
+                    </div>
+                    <div type="item">
+                        <div type="cable">
+                            <dateline>ST. PETERSBURG, October 31.</dateline>
+                            <p>A telegram yesterday from General Kuropatkin states the Japanese have
+                                begun to take the offensive.</p>
+                            <p>Admiral Alexeieff has left for Russia. (H.)</p>
+                        </div>
+                    </div>
+                    <div type="item">
+                        <head>JAPANESE WAR TAXATION.</head>
+                        <div type="cable">
+                            <dateline>TOKIO, October 31.</dateline>
+                            <p>The Diet will meet on the 25th November in order to discuss the
+                                Budget, which will double the Income Tax and augment other taxes for
+                                the purpose of meeting the war expenditure. (Reuter.)</p>
+                        </div>
+                    </div>
+                    <div type="item">
+                        <head>FIRST MANCHURIAN ARMY.</head>
+                        <div type="cable">
+                            <dateline>ST. PETERSBURG, October 31.</dateline>
+                            <p>General Linievitch has been appointed to command the first Manchurian
+                                Army. (R.)</p>
+                        </div>
+                    </div>
+                </div>
+                    <div type="advert" xml:id="deg-ad-bcp01">
+                        <head>BECK &amp; CO'S PILSENER BEER</head>
+                        <p>BREMEN.</p>
+                    <p>THE WORLD-WIDE KNOWN THE BEST.</p>
+                        <p>Obtainable from every Respectable Firm In Cairo, Alexandria &amp; the Sudan.</p>
+                        <p>Otherwise apply to</p>
+                        <p>V. J. FLEURENT, Cairo</p>
+                        <p>F. MICALLEF, Sole Agent, 11 Bab Midan, Alexandria</p>
+                    </div>
+                <cb n="2"/>
+                <div type="item">
```

Comment:
the following items need to be in the wire section, and there's a bit more tidying to be done.